### PR TITLE
(2982) ISPF reports templates are split between ODA and non-ODA

### PR DESCRIPTION
**Pull request by CristinaRO**
## Changes in this PR
- Upload templates for ISPF reports use the ODA type of the report

### Before
Screenshot 2023-11-22 at 14 10 09

### After
Screenshot 2023-11-22 at 14 09 20

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [x] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

**Comment by CristinaRO** [comment]
[ISPF reports need to only show the template corresponding to their ODA type](https://trello.com/c/hcSMGLgf/2982-ispf-reports-need-to-only-show-the-template-corresponding-to-their-oda-type)

File: app/controllers/activities/uploads_controller.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def update
     authorize report, :upload?
 
     @report_presenter = ReportPresenter.new(report)
-    @type = params[:type].to_sym
+    @type = @report_presenter.oda_type
```
Comment (mec) [inline app/controllers/activities/uploads_controller.rb:33]: Do we even need `@type`?

**Review by mec — APPROVED** [review]
One thing, but not worried. 🚢

File: app/controllers/activities/uploads_controller.rb (hunk 1 above)
Comment (CristinaRO) [reply app/controllers/activities/uploads_controller.rb:33]: You're right, we don't! 🎉

**Pull request merged**